tensorflow-serving/rockcraft.yaml

```diff
@@ -0,0 +1,117 @@
+# https://github.com/tensorflow/serving/blob/2.1.0/tensorflow_serving/tools/docker/Dockerfile
+name: tensorflow-serving
+summary: An image for Seldon Tensorflow Serving
+description: |
+  This image is used as part of the Charmed Kubeflow product.
+version: 2.13.0_20.04_1 # <upstream-version>_<base-version>_<Charmed-KF-version>
+license: Apache-2.0
+base: ubuntu:20.04
+run-user: _daemon_
+services:
+  tensorflow-serving:
+    override: replace
+    summary: "tensorflow-serving service"
+    startup: enabled
+    command: bash -c 'tensorflow_model_server --port=8500 --rest_api_port=8501 --model_name=${MODEL_NAME} --model_base_path=${MODEL_BASE_PATH}/${MODEL_NAME} "$@"'
+    environment:
+      MODEL_NAME: "model"
+      MODEL_BASE_PATH: "/models"
+platforms:
+  amd64:
+
+parts:
+  tensorflow-serving:
+    plugin: nil
+    source: https://github.com/tensorflow/serving/
+    source-type: git
+    source-tag: 2.13.0
+    build-packages:
+      - automake
+      - build-essential
+      - ca-certificates
+      - curl
+      - git
+      - libcurl3-dev
+      - libfreetype6-dev
+      - libpng-dev
+      - libtool
+      - libzmq3-dev
+      - mlocate
+      - openjdk-8-jdk
+      - openjdk-8-jre-headless
+      - pkg-config
+      - pip
+      - python-dev
+      - python3.8
+      - python3.8-dev
+      - python3-pip
+      - python3.8-venv
+      - software-properties-common
+      - swig
+      - tar
+      - unzip
+      - wget
+      - zip
+      - zlib1g-dev
+      - python3-distutils
+    build-environment:
+      - BAZEL_VERSION: "5.3.0"
+    override-build: |
+      set -e
+      python3.8 -m pip install pip --upgrade
+      update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 0
```

Comment:
do you really need this?

Reply:
Also, ensuring the default is set correctly.